### PR TITLE
Nerfs spiders defence stat against flyswatters - Alt  Australian crew buff.

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

https://www.youtube.com/watch?v=-ok-hwUFwHE

## Why It's Good For The Game

Flyswatters needs buff against spiders

## Changelog
:cl:
balance: Flyswatters now deals more damage to the spider clan, but not ninjas
/:cl:

### CODE REVIEW

**Review by Ghommie — CHANGES_REQUESTED** [review]
Alt title: Australian crew buff.

Giant spiders being one-shottable by flyswatters is a bit excessive though, would be more reasonable if it dealt an extra 15-25 damage instead.

**Comment by Trilbyspaceclone** [comment]
Please?

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -659,6 +662,11 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 			if(istype(target, /mob/living/))
 				var/mob/living/bug = target
 				bug.death(1)
+		if(is_type_in_typecache(target, spider_panic))
+			to_chat(user, "<span class='warning'>You easily land a critical blow on the [target].</span>")
+			if(istype(target, /mob/living/))
+				var/mob/living/bug = target
+				bug.adjustBruteLoss(-80)
```
Comment (silicons) [inline code/game/objects/items/weaponry.dm]: isn't this healing them

and what the hell no
Fun gimmicks are fun, hell I'd agree with 15-20 damage, but why the hell should this do the shotgun's equivalent of damage against spiders?

Comment (Trilbyspaceclone) [reply code/game/objects/items/weaponry.dm]: Spiders have over 100 hp is why, a welder deals around 23 iirc do to their armor values.
ill have it do 40

**Pull request merged**